openssh/patch-sandbox-darwin.c-apple-sandbox-named-external.diff

```diff
@@ -2,18 +2,18 @@
 +++ b/sandbox-darwin.c	2015-10-24 04:41:19.000000000 +0200
 @@ -62,8 +62,16 @@ ssh_sandbox_child(struct ssh_sandbox *bo
  	struct rlimit rl_zero;
- 
+
  	debug3("%s: starting Darwin sandbox", __func__);
 +#ifdef __APPLE_SANDBOX_NAMED_EXTERNAL__
 +#ifndef SANDBOX_NAMED_EXTERNAL
 +#define SANDBOX_NAMED_EXTERNAL (0x3)
 +#endif
-+	if (sandbox_init("@PREFIX@/share/openssh/org.openssh.sshd.sb",
++	if (sandbox_init("/System/Library/Sandbox/Profiles/org.openssh.sshd.sb",
```

Comment:
@mistydemeo I'm struggling to figure out how to make this patch conditional for Sierra.  However, my PR does not touch "org.openssh.sshd.sb" (and whether it should be "com.openssh.sshd.sb" which is in the original.  It only fixes the issue of "@PREFIX" which is definitely broken. The PR should be accepted because it clearly fixes a problem, even though it does not fix the org/com issue.